src/1.21.5/main/java/me.owdding/skyocean/mixins/DimensionSpecialEffectsMixin.java

```diff
@@ -0,0 +1,20 @@
+package me.owdding.skyocean.mixins;
+
+import com.llamalad7.mixinextras.injector.ModifyReturnValue;
+import me.owdding.skyocean.config.features.misc.MiscConfig;
+import net.minecraft.client.renderer.DimensionSpecialEffects;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+@Mixin(DimensionSpecialEffects.class)
+public class DimensionSpecialEffectsMixin {
+
+    @ModifyReturnValue(method = "getCloudHeight", at = @At("RETURN"))
```

Comment:
Change to modifying the return result of getCloudsType instead in a common mixin